bindings/nodejs/hawktracer_client_nodejs.hpp

```diff
@@ -54,7 +55,8 @@ class Client: public ObjectWrap<Client>
     public:
         bool is_started() const
         {
-            return static_cast<bool>(_client_context);
+            bool reader_connected = _client_context->get_reader_connected();
```

Comment:
This changes the semantics of `is_started()`. Before it was just checking the status, but not `get_reader_connected()` will try to connect. Better rename this or move `get_reader_connected()` to somewhere else. Why not in `start()`?

Reply:
Done